test/disyuntor_test.rb

```diff
@@ -0,0 +1,205 @@
+if ENV["COVERAGE"]
+  require "simplecov"
+  SimpleCov.start do
+    add_filter "/test/"
+    add_filter "/spec/"
```

Comment:
Given that you removed the `spec` folder, maybe you can remove this filter as well